holidays/financial/london_stock_exchange.py

```
@@ -0,0 +1,75 @@
+        * <https://en.wikipedia.org/wiki/Bank_holiday>
+        * [LSE business days](https://web.archive.org/web/20260617134450/https://www.londonstockexchange.com/equities-trading/business-days)
+    """
+
+    country = None  # type: ignore[assignment]
+    market = "XLON"
+    parent_entity = UnitedKingdom
+    parent_entity_subdivision_code = "ENG"
+    supported_categories = (HALF_DAY, PUBLIC)
+    start_year = 2000
+
+    def _populate_public_holidays(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_populate_public_holidays`

Add return type annotation: `None`

(ANN202)